docs/usage.md

```diff
@@ -71,4 +71,22 @@ urlpatterns = [
 ]
 ```
 
-See the [API documentation below](#api) for more details and examples.
+See the [API reference](/api/state) for more details and examples.
+
+## Defining flags
+
+### `FLAGS`
+
+Default: `{}`
+
+
+## Caching flag conditions
+
+It is possible that a request cycle could check numerous flags, or check a single flag multiple times. Because flag conditions created in the Django admin are stored in the database, this can result in multiple identical database queries. To cache flag conditions with a cache key of `flags_conditions`:
```

Comment:
isn't the cache key in this example just 'flags'?

Reply:
Yup. Corrected and expanded a bit about the defaults.